downstream/modules/automation-hub/proc-configure-content-signing-on-pah.adoc

```diff
@@ -15,7 +15,7 @@ To successfully sign and publish {CertifiedName}, you must configure {PrivateHub
 +
 [NOTE]
 ====
-This script will act as the signing service and must generate an ascii-armored detached `gpg` signature for that file using the key specified through the `PULP_SIGNING_KEY_FINGERPRINT` environment variable.
+This script acts as the signing service and must generate an ascii-armored detached `gpg` signature for that file using the key specified through the `PULP_SIGNING_KEY_FINGERPRINT` environment variable.
```

Comment:
I have a question about "the script... must generate". Could it potentially not do that? Is the user configuring this and ensuring that the script generates the signature?

Is this accurate:
the key specified through... > the key that you specified through...